litellm/proxy/client/cli/commands/keys.py

```diff
@@ -88,6 +88,8 @@ def list(
 @click.option("--spend", type=float, help="Maximum spend limit for this key")
 @click.option("--duration", type=str, help="Duration for which the key is valid (e.g. '24h', '7d')")
 @click.option("--key-alias", type=str, help="Alias/name for the key")
+@click.option("--key-name", type=str, help="Display/abbreviation name for the key (separate from key_alias)")
```

Comment:
can you add an example of this in your description eg. sk-12.....2oA

Reply:
Yep, check out 7eb128b90.